chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/attachmentUploadValidatorMiddleware.ts

```diff
@@ -58,8 +60,24 @@ const validateFileExtension = (attachment: any, allowedFileExtensions: string):
     return allExtensions.indexOf(fileExtension) > -1;
 };
 
-const validateFileSize = (attachmentSize: string, maxUploadFileSize: string): boolean => {
-    return (maxUploadFileSize && parseInt(maxUploadFileSize) * MBtoBRatio) > parseInt(attachmentSize);
+const validateFileSize = (attachmentSize: string, maxUploadFileSize: number): boolean => {
+    return (maxUploadFileSize * MBtoBRatio) > parseInt(attachmentSize);
+};
+
+const getmaxUploadFileSize = (maxFileSizeSupportedByDynamicsStr:string, contentType: string): number => {
```

Comment:
nit: Rename to `getMaxUploadFileSize`

Reply:
nit: spacing

`maxFileSizeSupportedByDynamicsStr:string` -> `maxFileSizeSupportedByDynamicsStr: string`

Reply:
done